pkg/util/overridemanager/labelannotationoverrider.go

```diff
@@ -47,6 +47,10 @@ func buildLabelAnnotationOverriderPatches(rawObj *unstructured.Unstructured, ove
 				continue
 			}
 		}
+		// RFC6902 https://datatracker.ietf.org/doc/html/rfc6902/#appendix-A.14
+		// Refer to a key with ~ or / in its name, you must escape the characters with ~0 and ~1 respectively.
+		// More info: https://jsonpatch.com/
```

Comment:
```suggestion
		// If the key contains '/', we must replace(escape) it to '~1' according to the
		// rule of jsonpath identifying a specific value.
		// See https://jsonpatch.com/#json-pointer for more details.
		// Note: here don't replace '~' because it is not a valid character for
		// both annotations and labels, the key with '~' should be prevented at
		// the validation phase.
```